src/VisualStudio/CSharp/Impl/Interactive/CSharpVsInteractiveWindowProvider.cs

```diff
@@ -58,8 +58,7 @@ internal sealed class CSharpVsInteractiveWindowProvider : VsInteractiveWindowPro
 
         protected override Guid Id => CSharpVsInteractiveWindowPackage.Id;
 
-        // Note: intentionally left unlocalized (we treat these words as if they were unregistered trademarks)
```

Comment:
Change is good to me, but is there any background information about this comment?

Reply:
I don't remember. Couldn't find anything. Even if we wanted to keep "Interactive" in English it should be in the loc file with explicit note.

Reply:
I found this https://github.com/dotnet/roslyn/pull/5808
@KevinH-MS @genlu  Do you remember why this is not localized?

Reply:
I don't remember either. From a pure functionality standpoint, I don't see why we don't want to localize it.